ad713x-fmcz/ad713x.h

```diff
@@ -643,45 +643,51 @@ typedef enum ad713x_supported_dev_ids {
 	ID_AD7136
 } ad713x_supported_dev_ids;
 
-typedef enum {
-	LOW_POWER = 0x01,
+typedef enum ad713x_power_mode {
```

Comment:
i don't know if we should use this intermediary step. This part is in its early stages so we can't just talk yet about backward compatibility. We should do it the right way from now on.